scenario/scenario.py

```diff
@@ -167,43 +172,28 @@ def _play_meta(
         # decompose the meta event
         events = []
 
-        if event.name == ATTACH_ALL_STORAGES:
+        if event.name in [ATTACH_ALL_STORAGES, DETACH_ALL_STORAGES]:
             logger.warning(f"meta-event {event.name} not supported yet")
             return
 
-        elif event.name == DETACH_ALL_STORAGES:
-            logger.warning(f"meta-event {event.name} not supported yet")
-            return
-
-        elif event.name == CREATE_ALL_RELATIONS:
-            if context:
-                for relation in context.relations:
-                    # RELATION_OBJ is to indicate to the harness_ctx that
-                    # it should retrieve the
-                    evt = Event(
-                        f"{relation.meta.endpoint}-relation-created",
-                        args=(
-                            InjectRelation(
-                                relation.meta.endpoint, relation.meta.relation_id
-                            ),
+        if event.name in [CREATE_ALL_RELATIONS, BREAK_ALL_RELATIONS] and context:
+            for relation in context.relations:
+                # RELATION_OBJ is to indicate to the harness_ctx that
+                # it should retrieve the
+                if CREATE_ALL_RELATIONS:
+                    name = f"{relation.meta.endpoint}-relation-created"
```

Comment:
this should be `event_name`, explicit is better than implicit, especially when there's so much going on